source/actions3/actions/CLSSIS3820ScalerDarkCurrentMeasurementActionInfo.cpp

```diff
@@ -0,0 +1,30 @@
+#include "CLSSIS3820ScalerDarkCurrentMeasurementActionInfo.h"
+
+CLSSIS3820ScalerDarkCurrentMeasurementActionInfo::CLSSIS3820ScalerDarkCurrentMeasurementActionInfo(CLSSIS3820Scaler *scaler, double secondsDwell, QObject *parent) :
+	AMListActionInfo3(QString(), QString(), QString(), parent)
```

Comment:
@helfrij, this action/actionInfo looks good other than my comment to @davidChevrier.  We may need to come up with a better way to get a hold of the scaler pointer, but everything looks fine.